README.md

```diff
@@ -54,12 +54,21 @@ With Standard Containers we can put an end to that embarrassment, by making INDU
 
 # Contributing
 
-Development happens on github for the spec.  Issues are used for bugs and actionable items and longer
-discussions can happen on the mailing list.  You can subscribe and join the mailing list on
-[google groups](https://groups.google.com/a/opencontainers.org/forum/#!forum/dev).
+Development happens on github for the spec.
+Issues are used for bugs and actionable items and longer discussions can happen on the [mailing list](#mailing-list).
 
-The specification and code is licensed under the Apache 2.0 license found in 
-the `LICENSE` file of this repository.  
+The specification and code is licensed under the Apache 2.0 license found in the `LICENSE` file of this repository.  
+
+## Discuss your design
+
+The project welcomes submissions, but please let everyone know what you are working on.
+
+Before undertaking a not-trivial change to this specification, send mail to the [mailing list](#mailing-list) to discuss what you plan to do.
+This gives everyone a chance to validate the design, helps prevent duplication of effort, and ensures that the idea fits.
+It also guarantees that the design is sound before code is written; a github pull-request is not the place for high-level discussions.
+
+Typos and grammatical errors can go straight to a pull-request.
+If there is question, start on the [mailing-list](#mailing-list).
```

Comment:
Otherwise it could read like you're pushing all questions (some of which might be better off as issues) to the mailing list.